arcgis10_mapping_tools/MapAction/MapAction/Utilities.cs

```diff
@@ -103,6 +104,37 @@ public static string createXML(Dictionary<string, string> usDict, string rootEle
 
         #endregion
 
+        #region Public method createXML
+        //Creates an xml given a dictionary of tags and values.  Also pass in the root element, file path and filename.
+        public static string createXML(MapAction.OperationConfig operationConfig, string path, string fileName)
+        {
+            //set output path and filename
+            string pathFileName;
+            //check to see if the path is the root of the directory, if so remove additional \
+            string root = System.IO.Path.GetPathRoot(path);

```

Comment:
Minor point, but wouldn't `Path.Combine(path, fileName)` handle the logic of working out whether or a directory separator is required and add if necessary? Would save a few lines here.